frontend/src/tw-components/index.tsx

```diff
@@ -1,3 +1,5 @@
 export { default as AuthNav } from "./AuthNav";
 export { default as HeaderNav } from "./HeaderNav";
-export { default as TextField } from "./TextField";
+export { default as InputGroup } from "./InputGroup";
```

Comment:
the `InputGroup` component was renamed to `TextField`

must have been accident with pulling down latest changes from main branch. I think just removing the line may work but this is what it looks like on our local machines right now

![image](https://github.com/hackforla/CivicTechJobs/assets/73561520/4103d303-e17c-4c9b-8d93-d792fbcd00e3)

Reply:
Yes, confirming that removing the InputGroup export works.

Reply:
InputGroup has been removed